core/ui/public/i18n/en/translation.json

```diff
@@ -544,10 +544,10 @@
     "samba_placeholder":"Share name",
     "smb_short": "SMB",
     "samba_tooltips":"Only shares with SMB2/3 protocols are supported. Deprecated SMB1/CIFS shares are not secure and cannot be accessed.",
-    "cluster": "Local storage",
-    "cluster_short": "Cluster",
+    "cluster_short": "Local storage",
     "node": "Node",
-    "cluster_placeholder": "Choose a node in the cluster"
+    "cluster_placeholder": "Choose a node in the cluster",
```

Comment:
The name of this i18n key is a bit too generic, let's rename it to something like `"choose_cluster_node"`, or let's remove it and use `common.choose`, you choose

Reply:
https://github.com/search?q=repo%3ANethServer%2Fns8-core%20cluster_placeholder&type=code

we have a function that return all the same, do you want we change things for all ?

```
    selectedProviderHelper() {
      if (this.isBackblazeSelected) {
        return "backblaze_placeholder";
      } else if (this.isAmazonS3Selected) {
        return "aws_placeholder";
      } else if (this.isGenericS3Selected) {
        return "s3_placeholder";
      } else if (this.isAzureSelected) {
        return "azure_placeholder";
      } else if (this.isSambaSelected) {
        return "samba_placeholder";
      } else if (this.isClusterSelected) {
        return "cluster_placeholder";
      } else {
        return "url_placeholder";
      }
    },
```

Reply:
Didn't notice it, it's ok then :v: